lib/commands/envs.ts

```diff
@@ -23,14 +24,7 @@ import * as cf from '../utils/common-flags';
 import { getBalenaSdk, getVisuals, stripIndent } from '../utils/lazy';
 import { applicationIdInfo } from '../utils/messages';
 
-interface FlagsDef {
-	fleet?: string;
-	config: boolean;
-	device?: string; // device UUID
-	json: boolean;
-	help: void;
-	service?: string; // service name
-}
+type FlagsDef = Interfaces.InferredFlags<typeof EnvsCmd.flags>;
```

Comment:
@thgreasi @myarmolinsky  Also here is one example in case we need to create a function that receives the parsed type of the flags we can use the new (and very userfull) `Interfaces.InferredFlags` instead of manually mantaining types, we let oclif do the dirty job for us.